.github/workflows/build.yaml

```diff
@@ -36,24 +37,26 @@ jobs:
         # any additional builds for windows and other python versions are
         # handled via `include` to avoid an over-large test matrix
         os: [ubuntu-latest, macos-latest]
-        python-version: [2.7, 3.5, 3.9]
+        python-version: ["2.7", "3.5", "3.10"]
```

Comment:
Thanks for converting everything to strings. Saves us the inevitable heartache when 3.20 rolls around! 😉